Makefile

```diff
@@ -77,6 +77,14 @@ test: smart-contract-test ## run unit tests for all packages, or a single packag
 smoke-tests: ## run smoke tests
 	source .venv/bin/activate && python3 -m pytest tests/
 
+.PHONY: stress-test-local-swarm
+stress-test-local-swarm: ## run stress tests on a local node swarm
+	source .venv/bin/activate && \
+		python3 -m pytest tests/test_stress.py \
+		--stress-request-count=10000 \
```

Comment:
Fails if the total request couldn't be completed in the 60s timeout, without showing results

Reply:
Reduce to 3000